azerothcore/azerothcore-wotlk
# fix(Core/Player): Prevent looting already looted items

* https://github.com/TrinityCore/TrinityCore/commit/586c00fe2e226da99968ebe763d3226c6134658c

Co-Authored-By: Gildor <521036+jildor@users.noreply.github.com>





## Changes Proposed:
-  Prevent looting already looted items

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4415



## Tests Performed:
- Builds



## How to Test the Changes:

Try steps in https://github.com/azerothcore/azerothcore-wotlk/issues/4415 or

1. get two players in a group
2. .quest add 47 (in two players)
3. .go cre id 475
4. kill npc
5. click with two players the npc to see loot
6. do .reload all for lag server (you can do 2-3 times in a row)
7. click with two players the quest item before reload all finish
8. two players get the item


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
